ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -567,6 +567,8 @@ + (void)updateViewController:(UIViewController *)vc
 #endif
   }
 #if !TARGET_OS_TV
+  // Workaround for the wrong rotation of back button arrow in RTL mode.
+  navitem.hidesBackButton = true;
   navitem.hidesBackButton = config.hideBackButton;
```

Comment:
Hey, would you mind describing why do we first set `navitem.hidesBackButton = true` and then immediately we set it to the `config.hideBackButton`? Whats the mechanism here?

Reply:
That's because when you're setting your application to RTL mode the arrow in the back button is still pointing to the left - which is wrong, because in RTL mode you should have an arrow pointing to the right. It seems that hiding back button for a while fixes this issue, because any update related to the back button modification updates the it. To be fair I don't know why this is working, unfortunately changing the `semanticContentAttribute` in `dispatch_after` doesn't fix that either.